reana/reana_benchmark/cli.py

```diff
@@ -210,3 +211,20 @@ def collect_command(workflow: str, force: bool) -> NoReturn:
         collect(workflow, force)
     except Exception as e:
         logger.error(f"Something went wrong when collecting results: {e}")
+
+
+@reana_benchmark.command(name="monitor")
+@workflow_option
+@click.option(
+    "--sleep",
+    "-s",
+    help="Sleep in seconds between collecting metrics, default 30",
```

Comment:
Very minor: I guess to be consistent with `reana-client` and `reana-dev` commands format we could use 
```
Sleep in seconds between collecting metrics [default=30]
```
`help` description here, WDYT?

Reply:
Done.